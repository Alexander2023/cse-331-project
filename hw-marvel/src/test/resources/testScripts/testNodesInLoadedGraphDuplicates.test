# Tests that duplicates in csv file are not added to the loaded graph

LoadGraph smallWithDuplicates smallWithDuplicates.csv

ListNodes smallWithDuplicates