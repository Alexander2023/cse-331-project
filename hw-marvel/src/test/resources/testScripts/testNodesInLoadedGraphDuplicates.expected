# Tests that duplicates in csv file are not added to the loaded graph

loaded graph smallWithDuplicates

smallWithDuplicates contains: c1 c2